cli_reference/openshift_developer_cli/debugging-applications-in-odo.adoc

```diff
@@ -0,0 +1,23 @@
+[id='debugging-applications-in-odo']
+= Debugging applications in `{odo-short}`
+include::modules/openshift-developer-cli-attributes.adoc[]
+include::modules/common-attributes.adoc[]
+:context: debugging-applications-in-odo
+toc::[]
+
+:FeatureName: Interactive debugging in `{odo-short}` 
+include::modules/technology-preview.adoc[leveloffset=+1]
+
+With `{odo-short}`, you can attach a debugger to remotely debug your application. This feature is only supported for NodeJS and Java components.
```

Comment:
All of these instances of "odo-short" should be "odo-title" according to your attributes file. They're not being replaced. Just a reminder to review the built output (a preview is auto generated when you open a PR against master) and you can see them.